### ISSUE TITLE
React is not defined when using 8.1.1

### ISSUE THREAD

**Issue by pkeuter** [LouisBrunner/dnd-multi-backend#338]
First of all: thank you for this useful package. 

After installing version 8.1.1, I get `Uncaught ReferenceError: React is not defined`. It's thrown from [this file](https://github.com/LouisBrunner/dnd-multi-backend/blob/main/packages/react-dnd-multi-backend/src/index.ts). I am using vite  6.0.3 and React 19. When reverting back to 8.0.3 everything works fine.

When I compare the 2 output files, I can see that the import of React is now gone:

8.0.3:
```
export * from "dnd-multi-backend";
import i, { useState as l, createContext as D } from "react";
import { DndProvider as m } from "react-dnd";
import { MultiBackend as P } from "dnd-multi-backend";
```
8.1.1:
```
export * from "dnd-multi-backend";
import { MultiBackend as d } from "dnd-multi-backend";
import { createContext as l, useState as D } from "react";
import { DndProvider as m } from "react-dnd";
```

Where the `import i ... from "react"` is the import that is now missing. I suspect this is the change that is causing the issue:

https://github.com/LouisBrunner/dnd-multi-backend/compare/v8.0.3...v8.1.1#diff-54e58e979d05db6de025630d7251a002b017d0bc7ff9cd9bcd22663f3b68f8b5R2

**Comment by LouisBrunner** [comment]
Hi @pkeuter,

Thanks for your report.

Yes i have changed some of the build scripts when releasing v8.1.0 and did encounter some weird issues but I didn't know it would propagate to users. Sorry about that.

I will look into it now.

**Comment by LouisBrunner** [comment]
I released v8.1.2 which should address this issue. Can you confirm if this is resolved or not?

**Comment by pkeuter** [comment]
Thank you for the quick fix @LouisBrunner, it has been resolved. I will close this issue.
